Python/pystate.c

```diff
@@ -34,55 +35,66 @@ to avoid the expense of doing their own locking).
 extern "C" {
 #endif
 
-int _PyGILState_check_enabled = 1;
+void
+_PyRuntimeState_Init(_PyRuntimeState *runtime)
+{
+    _PyRuntimeState initial = {};
```

Comment:
Empty initializers aren't standard. You should probably just `memset` it to 0.

Reply:
fixed